programs/buck_version.py

```diff
@@ -34,12 +34,20 @@ def fileno(self):
 
 def is_git(dirpath):
     dot_git = os.path.join(dirpath, '.git')
-    return all([
-        os.path.exists(dot_git),
-        os.path.isdir(dot_git),
-        which('git'),
-        sys.platform != 'cygwin',
-    ])
+    if(which('git') and sys.platform != 'cygwin'):
+        if(os.path.exists(dot_git) and os.path.isdir(dot_git)):
```

Comment:
In Python code we generally don't use parentheses around the condition in if statements, please remove those here and on the line above.